src/transport/PASESession.cpp

```diff
@@ -546,28 +546,25 @@ CHIP_ERROR PASESession::SendMsg1()
     uint8_t X[kMAX_Point_Length];
     size_t X_len = sizeof(X);
 
-    System::PacketBufferHandle msg_pA;
+    ReturnErrorOnFailure(mSpake2p.BeginProver(reinterpret_cast<const uint8_t *>(""), 0, reinterpret_cast<const uint8_t *>(""), 0,
```

Comment:
not this PR, but wondering why we do this cast ... maybe supporting passing nullptr would be clearer.

Reply:
Looks like it's already supported. I'll update the code.